pkg/utils/helper.go

```diff
@@ -312,6 +360,16 @@ func GetResourceList(kubeCmds []string, separator string) ([]string, error) {
 	return strings.Split(strings.Trim(resourceList, "\""), separator), nil
 }
 
+//contains checks if an array contains a value
+func Contains(flagsList []string, flag string) bool {
```

Comment:
💡 Not able to follow this moving around of code because previously this funct was privately used in the main `aks-perisceop.go` file